### PR TITLE
modify some errors of plurality

### PR DESCRIPTION
**Pull request by liliVicky**
Signed-off-by: liwenqi <vikilwq@zju.edu.cn>

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
Please consider squashing https://github.com/docker/docker/pull/29620

File: daemon/cluster/cluster.go [hunk 1]
```diff
@@ -287,7 +287,7 @@ func (c *Cluster) Init(req types.InitRequest) (string, error) {
 	// will be used as local address, if it belongs to this system.
 	// If the advertise address is not local, then we try to find
 	// a system address to use as local address. If this fails,
-	// we give up and ask user to pass the listen address.
+	// we give up and ask users to pass the listen address.
```
Comment (dnephin) [inline daemon/cluster/cluster.go]: "ask the user"

File: daemon/events/events.go [hunk 2]
```diff
@@ -78,7 +78,7 @@ func (e *Events) Evict(l chan interface{}) {
 	e.pub.Evict(l)
 }
 
-// Log broadcasts event to listeners. Each listener has 100 millisecond for
+// Log broadcasts event to listeners. Each listener has 100 milliseconds for
```
Comment (dnephin) [inline daemon/events/events.go]: "Log a broadcast event to listeners. Each listener has 100 milliseconds to receive the event ..."

**Review by dnephin — COMMENTED** [review]
Thanks! A couple suggestions

File: daemon/cluster/cluster.go (hunk 1 above)
Comment (liliVicky) [inline daemon/cluster/cluster.go]: Thank you for your advice, I have modified. @dnephin

**Review by dnephin — APPROVED** [review]
LGTM

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by cpuguy83** [comment]
Windows CI is still pending, but this doesn't effect anything CI related, and lint/vet validations have all passed.

Merging.

**Pull request merged**